schedule-pod-restart/README.md

```diff
@@ -0,0 +1,48 @@
+# Cronjob Pod Restart
+
+Automatically restarting pods with labels based on a cron schedule.
+
+## TL;DR
+
+```console
+helm repo add coditeu https://coditeu.github.io/charts
+helm repo update
+helm install coditeu/cronjob-pod-restart
+```
+
+## Introduction
+
+## Installing the Chart
+
+To install the chart with the release name `schedule-pod-restart`:
+
+```console
+$ helm install schedule-pod-restart coditeu/schedule-pod-restart
+```
+
+## Uninstalling the Chart
+
+To uninstall the `schedule-pod-restart` deployment:
+
+```console
+helm uninstall schedule-pod-restart
+```
+
+The command removes all the Kubernetes components associated with the chart and
+deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the chart and
+their default values.
+
+| Parameter                    | Description                                                              | Default                           |
+|:-----------------------------|:-------------------------------------------------------------------------|:----------------------------------|
+| `nameOverride`               | Name of the cronjob that will be created                                 | Chart Name `schedule-pod-restart` |
+| `serviceAccount.name`        | Name of the service account to use                                       | `operations-pod-restart`          |
+| `serviceAccount.annotations` | Annotations for the service account                                      | None                              |
+| `role.name`                  | Name of the role to create                                               | `manage-pods-role`                |
+| `bindings.role.name`         | Name of the role binding                                                 | `manage-pods-role-binding`        |
+| `cronjob.schedule`           | A cronjob expression that defines the schedule when the job should run   | `0 0 * * *`                       |
```

Comment:
Maybe add a link to https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#cron-schedule-syntax?

```suggestion
| `cronjob.schedule`           | A cronjob expression that defines the schedule when the job should run. Learn more about the format [in the official documentation](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#cron-schedule-syntax).   | `0 0 * * *`                       |
```